package.json

```diff
@@ -121,5 +121,6 @@
       "import": "./dist/utils/index.js",
       "types": "./dist/utils/index.d.ts"
     }
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
What if you used yarn 1.21? I believe I have this version and 1. I never use core pack 2. I never have to add this to package.json

Reply:
yarn 1.21 works as well, I'm not sure why it did that to me but everytime i tried removing the `packageManager` line and committing it would throw an error about `Unstaged changes could not be restored due to a merge conflict!`.

Reply:
I never had this problem so I can't help you. 

Try reinitializing your yarn and remove core pack. Seems to introduce more problems than it fixes.

Reply:
changing yarn versions to `1.12.1` seems to have fixed the problem! pushed a new commit to remove the packageManager line from `package.json`